feature: enable loading 1password secrets from file

wcarlsen opened the pull request:
I hope this approach with loading 1password environment variables from an 1password specific env file can solve below feature request.

Fixes #93

wcarlsen commented:
Okay I will fix my signing setup, which apparently is broken beyond repair right now.

wcarlsen commented on README.md:
```diff
@@ -63,6 +64,7 @@ jobs:
         env:
           OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
           SECRET: op://app-cicd/hello-world/secret
+          OP_ENV_FILE: "path/to/.env.tpl"
```
I added a reference to the new test/.env.tpl as a comment, to not blow up the already nice, short and simple documentation. Let me know if you want the full blown versions?

volodymyrZotov commented:
/ok-to-test https://github.com/1Password/load-secrets-action/commit/1850a6b4876c1ab27e3ed994fd2404e351c41e5a

volodymyrZotov commented:
@wcarlsen Thank you for addressing the comments!

FYI, I could not run tests agains your forked branch due to the issue on the [github action](https://github.com/peter-evans/slash-command-dispatch/issues/396) we use in the workflow.

I'll test it manually and return back.

wcarlsen commented:
> @wcarlsen Thank you for addressing the comments!
> 
> 
> 
> FYI, I could not run tests agains your forked branch due to the issue on the [github action](https://github.com/peter-evans/slash-command-dispatch/issues/396) we use in the workflow.
> 
> 
> 
> I'll test it manually and return back.

@volodymyrZotov damn 😀. Let me know if I can do anything to help.

volodymyrZotov commented:
@wcarlsen Unfortunately, this implementation it doesn't work. `process.loadEnvFile()` method was added in `v21.7.0`.
But the action uses `node20` which is latest stable `v20.x`.

I'd suggest to use `dotenv` for this feature.
The snippt below worked for me with node `v20.x`:
```
const dotenv = require('dotenv');
dotenv.config({ path: './.env' });

console.log(process.env.SECRET)
console.log(process.env.SECRET_IN_SECTION)
console.log(process.env.MULTILINE_SECRET)
```
